driver/pace/driver/report.py

```diff
@@ -0,0 +1,146 @@
+import copy
+import dataclasses
+import json
+from datetime import datetime
+from typing import Any, Dict, List
+
+import numpy as np
+
+from pace.util.mpi import MPI
+
+
+@dataclasses.dataclass
+class Experiment:
+    dataset: str
+    format_version: int
+    git_hash: str
+    timestamp: str
+    timesteps: int
+    version: str
```

Comment:
Version is a strange naming choice here (especially with format version present). Why is this not called backend?

Reply:
Changed to backend, I was trying to not change anything so we didn't have to update the viz plan. But looks like we would need to anyways because I can't use `hash` and had to change it to `git_hash`.